### DISCUSSION TITLE
Welcome to MultiversX Discussions!

### DISCUSSION THREAD

**Discussion by schimih** [ElrondNetwork/community#1] [Announcements]
### Welcome to the home of the MultiversX Devs Community!
If you are a developer, you came to the right place! :fire:
Here you can ask questions, share ideas, and build connections with each other—all right next to the MultiversX team. We hope that by using GitHub Discussions we enable **healthy and productive software collaboration**, and developers from the community can arise and become visible.

What do we want to achieve?
**1. Dedicated space for conversations;**
- mark quality responses and make the best answer more discoverable for future community members.
- thread conversations.
- create community polls;

**2. A right context for each conversation;**
- transform Github Discussions  into a dedicated space for the community to come together, learn, ask and answer questions, and have open-ended conversations with the MultiversX team;
- each conversation can be transformed into a future feature on the MultiversX Public Roadmap (see Community Disclaimer).

**3. Mobility**
 - ask and respond on-the-go, stay connected whenever and wherever is convenient for you.
 
###  **Living Our Values for DEVELOPERS**
 We are committed to maintaining a safe and healthy developers community. All contributors to this community are required to act with **integrity, show respect and build trust.** Only discussions inviting to discovery, development and value building will be promoted and will stay visible. Any topics violating the code of conduct or that are not approaching fair and honest subjects will be deleted.
 
 For any subject outside the developers area (EGLD value, product issues, opinions, trading trends etc.) you may use our telegram channels where our support team and community will be more than happy to support you.
 
**Stay Hungry! Stay Foolish!** :fire: :100:

**Comment by michavie** [comment]
you guys have really taken it to another level with the details & transparency for devs gained with the Github discussions tab, the 'releases' page on elrond.com/releases, and the public roadmap (which isn't quite that public yet – try with incognito: https://github.com/orgs/ElrondNetwork/projects/4)

a great step forward! ⚡️

**Reply by schimih** [reply]
Hey @michavie  ,
thanks for your message.

The roadmap is still private until we will have a first item coming from the community to plan. Hopefully soon!

**Comment by praulins** [comment]
I love this format. This forum must be listed in https://linktr.ee/multiversx.
